### PR TITLE
updated NEWTS group.html to include a brief bio and placeholder photo

### PR DESCRIPTION
**Pull request by andrewjjolly**

### CODE REVIEW

File: group.html [hunk 1]
```diff
@@ -88,6 +88,20 @@ <h3>Ryan Morris
 												<li><a href="http://ryanramorris.space" class="button">Website</a></li>
 											</ul>
 										</article>
+										<article>
+											<a href="#" class="image"><img src="images/aj_photo_1.jpg.jpg" alt="" /></a>
```
Comment (benmontet) [inline group.html]: Do you mean to have the filename be *.jpg.jpg?

File: group.html [hunk 2]
```diff
@@ -88,6 +88,20 @@ <h3>Ryan Morris
...
 										</article>
+										<article>
+											<a href="#" class="image"><img src="images/aj_photo_1.jpg.jpg" alt="" /></a>
+											<h3>Andrew Jolly
+											<a href="https://twitter.com/iamstillandrew" class="icon brands fa-twitter"><span class="label">Twitter</span></a>
+											<a href="https://github.com/andrewjjolly" class="icon brands fa-github"><span class="label">Github</span></a>
+											<a href="mailto:andrew.j.jolly@unsw.edu.au" class="icon fa-envelope"><span class="label">Email</span></a>
+											</h3>
+											<p>I am a first year physics PhD student at UNSW in Sydney. I am interested in utilizing machine learning techniques to
+												build accurate models of stellar surfaces and then use those models to find exoplanets in radial velocity observations.
```
Comment (benmontet) [inline group.html]: To be pedantic: you're going to find the exoplanets in radial veocity datasets, the observations are the moment of collecting the photons at the telescope.

**Review by benmontet — CHANGES_REQUESTED** [review]

File: group.html (hunk 1 above)
Comment (andrewjjolly) [reply group.html]: I did not.

File: group.html (hunk 2 above)
Comment (andrewjjolly) [reply group.html]: Ah yes. Not that pedantic to be fair.

**Pull request merged**